main.go

```diff
@@ -46,6 +46,12 @@ func main() {
 	e.DELETE("/waypoint/:waypointId", controllers.DeleteWaypoint)
 	e.GET("/waypoints", controllers.GetAllWaypoints)
 
+	//Ground Object
+	e.POST("/ground_object", controllers.CreateGroundObject)
+
+	//Payload
+	e.POST("/payload", controllers.CreatePayload)
+
```

Comment:
Methods to edit, update and delete Payload's also need to be added

Reply:
Same thing as Ground Object (added methods but no documentation as of yet)

Reply:
Added documentation.